### ISSUE TITLE
British Airways

### ISSUE THREAD

**Issue by TheRealJackUK** [kolos26/GEOFS-LiverySelector#115]
I made British Airways livery for A-380, also if it ain't good enough please make your own
![british airways a-380](https://user-images.githubusercontent.com/79092703/197613301-114b701c-4dfb-40e7-8446-442b3ab262ca.jpg)

**Comment by Egan-31** [comment]
nice, but this is just that there is already the british airways a380 in the livery selector

**Comment by kolos26** [comment]
Yes, sadly the british one aready exists, but if it's okay for you we should turn in to something like this:
![image](https://user-images.githubusercontent.com/77933334/198063225-7e72e480-6509-485c-97e0-1be1e1f0fbf7.png)
@Egan-31 We can do the same with your china soutern

**Comment by TheRealJackUK** [comment]
> Yes, sadly the british one aready exists, but if it's okay for you we should turn in to something like this: ![image](https://user-images.githubusercontent.com/77933334/198063225-7e72e480-6509-485c-97e0-1be1e1f0fbf7.png) @Egan-31 We can do the same with your china soutern

Sure
Btw soon I'm gonna make an fictional airline
Once I start operations is it ok to make a livery for it?
Also do players without the extension see liveries that use the extension?
